### PR TITLE
GitOps: Fix markdown.

### PR DESCRIPTION
**Pull request by Gacko**

### CODE REVIEW

File: src/content/advanced/gitops/apps/update_appcr.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ export APP_NAME=APP_NAME
     cd management-clusters/${MC_NAME}/organizations/${ORG_NAME}/workload-clusters/${WC_NAME}/mapi/apps/${APP_NAME}
     ```
 
-1. Edit the `appcr.yaml` if you want to update the App CR fields, like version, catalog, etc. For all the supported fields reference [the App CRD schema](https://docs.giantswarm.io/ui-api/management-api/crd/apps.application.giantswarm.io/)
+2. Edit the `appcr.yaml` if you want to update the App CR fields, like version, catalog, etc. For all the supported fields reference [the App CRD schema](https://docs.giantswarm.io/ui-api/management-api/crd/apps.application.giantswarm.io/)
```
Comment (LolloneS) [inline src/content/advanced/gitops/apps/update_appcr.md:46]:
```suggestion
2. Edit the `appcr.yaml` if you want to update the App CR fields, like version, catalog, etc. For all the supported fields reference [the App CRD schema](https://docs.giantswarm.io/use-the-api/management-api/crd/apps.application.giantswarm.io/)
```

File: src/content/advanced/gitops/apps/update_appcr.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ export APP_NAME=APP_NAME
     cd management-clusters/${MC_NAME}/organizations/${ORG_NAME}/workload-clusters/${WC_NAME}/mapi/apps/${APP_NAME}
     ```
 
-1. Edit the `configmap.yaml` if you want to update a non-secret user configuration
+2. Edit the `configmap.yaml` if you want to update a non-secret user configuration
```
Comment (LolloneS) [inline src/content/advanced/gitops/apps/update_appcr.md:56]: I think using `1.` for each line is default markdown, doesn't it work?

Comment (LolloneS) [reply src/content/advanced/gitops/apps/update_appcr.md:56]: But anything works for me honestly

**Review by LolloneS — CHANGES_REQUESTED** [review]

File: src/content/advanced/gitops/apps/update_appcr.md (hunk 2 above)
Comment (Gacko) [reply src/content/advanced/gitops/apps/update_appcr.md:56]: It should work but our CI complains.

**Review by LolloneS — APPROVED** [review]

**Pull request merged**